lib/improver/tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -163,6 +172,15 @@ def test_raises_error(self):
         with self.assertRaisesRegexp(IOError, msg):
             plugin.check_input_cubes(cubes)
 
+    def test_incorrect_units(self):
+        """Test that check_input_cubes method raises error if units are
+        different to the input cube."""
+        plugin = WeatherSymbols()
+        msg = 'Weather Symbols input cubes have threshold units'
```

Comment:
Adding the second line  "that diff from those required" would make this test more understandable

Reply:
:white_check_mark: